repo/packages/D/dcos-autoscaler/0/package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
@ryadav88  done .. please review again
here is the pull request for the examples repo https://github.com/dcos/examples/pull/170